packages/garbo/src/tasks/post/post.ts

```diff
@@ -0,0 +1,272 @@
+import {
+  availableChoiceOptions,
+  canAdventure,
+  cliExecute,
+  itemAmount,
+  mallPrice,
+  myAdventures,
+  myLevel,
+  myLocation,
+  putCloset,
+  reverseNumberology,
+  runChoice,
+  totalFreeRests,
+  use,
+  useSkill,
+  visitUrl,
+} from "kolmafia";
+import {
+  $effect,
+  $item,
+  $items,
+  $location,
+  $skill,
+  AutumnAton,
+  CinchoDeMayo,
+  clamp,
+  FloristFriar,
+  get,
+  getRemainingStomach,
+  have,
+  JuneCleaver,
+} from "libram";
+import { GarboStrategy, Macro } from "../../combat";
+import { globalOptions } from "../../config";
+import { computeDiet, consumeDiet } from "../../diet";
+import {
+  bestJuneCleaverOption,
+  freeRest,
+  juneCleaverChoiceValues,
+  valueJuneCleaverOption,
+} from "../../lib";
+import { teleportEffects } from "../../mood";
+import { GarboTask } from "../engine";
+import { Quest } from "grimoire-kolmafia";
+import bestAutumnatonLocation from "../../resources/autumnaton";
+import { estimatedGarboTurns, remainingUserTurns } from "../../turns";
+import { acquire } from "../../acquire";
+import { garboAverageValue } from "../../garboValue";
+import workshedTasks from "./worksheds";
+
+const STUFF_TO_CLOSET = $items`bowling ball, funky junk key`;
+function closetStuff(): GarboTask {
+  return {
+    name: "Closet Stuff",
+    completed: () => STUFF_TO_CLOSET.every((i) => itemAmount(i) === 0),
+    do: () => STUFF_TO_CLOSET.forEach((i) => putCloset(itemAmount(i), i)),
+    spendsTurn: false,
+  };
+}
+
+const BARF_PLANTS = [
+  FloristFriar.StealingMagnolia,
+  FloristFriar.AloeGuvnor,
+  FloristFriar.PitcherPlant,
+];
+function floristFriars(): GarboTask {
```

Comment:
Since we're going to use functions to return these, I would prefer to fully exclude post tasks that will never be applicable. 

I'd recommend something like making all these return 
```ts
GarboTask & { allowed: () => boolean } 
```
For Florist Plants
```
{
allowed: () => FloristFriar.have()
}
```

and then later on, just doing a filter
```ts
tasks: [
// ...
].filter((t) => t.allowed())
```
This would let our "post combat" engine be lighter for people with fewer resources.

Reply:
Done. Got to do some neat horrible type bullshit, too